cfme/networks/views.py

```diff
@@ -524,6 +527,81 @@ def is_displayed(self):
                 self.title.text == '{name} (Summary)'.format(name=self.context['object'].name))
 
 
+class FloatingIPToolbar(View):
+    """ Represents toolbar of floating ip section """
+    configuration = Dropdown("Configuration")
+    policy = Dropdown("Policy")
+    download = Dropdown("Download")
+    view_selector = View.nested(ItemsToolBarViewSelector)
+
+
+class FloatingIPDetailsToolbar(View):
+    """ Represents details toolbar of floating ip """
+    configuration = Dropdown("Configuration")
+    policy = Dropdown("Policy")
+    download = Button(title="Download summary in PDF format")
+
+
+class FloatingIPSideBar(View):
+    """ Represents left side bar, usually contains navigation, filters, etc """
+    pass
```

Comment:
Because for every view of that type exists one sidebar visible in ui, but currently it's empty. If they add some contents to it in future, in can be added there.